features/google/src/views/explore/explore.jsx

```diff
@@ -37,11 +45,44 @@ const ReportCard = () => {
 
 const ExploreView = () => {
     const { account } = useContext(PolyImportContext);
-    const { reportIsSent } = useContext(GoogleContext);
+    const { reportIsSent, handleReportSent } = useContext(GoogleContext);
 
     const history = useHistory();
     const exploreRef = useRef();
 
+    const handleCloseReportResult = () => {
+        handleReportSent(null);
+    };
+
+    const renderReportResult = () =>
+        reportIsSent !== null && (
+            <PopUpMessage
+                reportResultAnswer={
+                    reportIsSent ? " successfully" : " unsuccessfully"
+                }
+            >
+                {reportIsSent ? (
+                    <>
+                        <div>{i18n.t("explore:report.success")}</div>
+                        <img
+                            src="./images/close_green.svg"
+                            alt="close"
+                            onClick={handleCloseReportResult}
+                        />
+                    </>
+                ) : (
+                    <>
+                        <div>{i18n.t("explore:report.error")}</div>
+                        <img
+                            src="./images/close_red.svg"
```

Comment:
Have these been revised for accessibility? TBH, they look almost the same even to me...

Reply:
I'm not sure about that, that is something we could check with the designers. I will ask them, but in the meantime, I think this could work (it looks the same as in the Facebook feature).